core/src/main/java/hudson/Launcher.java

```diff
@@ -1240,9 +1240,9 @@ public Launcher getInner() {
 
     public static class IOTriplet implements Serializable {
         @Nonnull
-        InputStream stdout,stderr;
+        public InputStream stdout,stderr;
```

Comment:
I would prefer to have getter's. In case it's an immutable class, put also constructor otherwise setter's.
At least propose a constructor and put the fields as final.

-0.5